### PR TITLE
Menu section

### LINKED ISSUE #13: Menu Access
As a parent, I want access to menus.

### PR DESCRIPTION
**Pull request by PaulRoh97**
Implemented a menu section with filler menu item values

### CODE REVIEW

File: Covid Lunches/express/express-bootstrap/public/js/main.js [hunk 1]
```diff
@@ -158,6 +158,30 @@
     return false;
   });
 
+  // Picture Menu isotope and filter
+  $(window).on('load', function() {
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/public/js/main.js:162]: nice

File: Covid Lunches/express/express-bootstrap/views/index.njk [hunk 2]
```diff
@@ -73,7 +73,8 @@
                     <li class="active"><a href="/">Home</a></li>
                     <li><a href="#about">About</a></li>
                     <li><a href="#resources">Resources</a></li>
-                    <li><a href="#events">Pickup</a></li>
+                    <li><a href="#picture-menu">Menu</a></li>
+                    <!-- <li><a href="#events">Pickup</a></li> -->
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/views/index.njk:77]: potential source of future merge conflicts

**Review by F-421 — APPROVED** [review]
Nice mobile mode viewing.

**Review by abel-asfaw — APPROVED** [review]
Menu looks great!

**Pull request merged**